public/css/main.css

```diff
@@ -1,20 +1,207 @@
 @font-face {
   font-family: 'squealerregular';
   src: url('fonts/squealer-webfont.eot');
-  src: url('fonts/squealer-webfont.eot?#iefix') format('embedded-opentype'),
-    url('fonts/squealer-webfont.woff2') format('woff2'),
-    url('fonts/squealer-webfont.woff') format('woff'),
-    url('fonts/squealer-webfont.ttf') format('truetype'),
-    url('fonts/squealer-webfont.svg#squealerregular') format('svg');
+  src: url('fonts/squealer-webfont.eot?#iefix') format('embedded-opentype'), url('fonts/squealer-webfont.woff2') format('woff2'), url('fonts/squealer-webfont.woff') format('woff'), url('fonts/squealer-webfont.ttf') format('truetype'), url('fonts/squealer-webfont.svg#squealerregular') format('svg');
   font-weight: normal;
   font-style: normal;
 }
 
 body {
   background-image: radial-gradient(circle, #3241a6 0, #202a6b 120%);
-  font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI",
-    "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue",
-    "Arial", sans-serif;
+  font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue", "Arial", sans-serif;
+}
+
+
+/*Style group one stars*/
+
+.stars {
+  background: transparent url(../img/stars.png) repeat top center;
+  z-index: -1;
+  animation: moveStars 15s linear infinite;
+  -ms-animation: moveStars 15s linear infinite;
+  -moz-animation: moveStars 15s linear infinite;
+  -webkit-animation: moveStars 15s linear infinite;
+}
+
+
+/*Had to add more stars and offset them*/
+
+.starsTwo {
```

Comment:
Would you mind changing these classes to hyphens rather than camel case? So `stars-two`, `stars-three` etc.?